hack/testExamplesE2EAgainstKapply.sh

```diff
@@ -7,7 +7,7 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-mdrip --blockTimeOut 60m0s --mode test \
+mdrip --blockTimeOut 6m0s --mode test \
```

Comment:
6 minutes!?

Reply:
Yes. We can increase it as we add more tests. This helps us fail fast in cases like deployments not reaching current state etc. These tests should be timed precisely.